theseus/__init__.py

```diff
@@ -76,3 +76,8 @@
 from .theseus_layer import TheseusLayer
 
 import theseus.embodied as eb
+
+# Aliases for some standard cost functions
+Difference = eb.VariableDifference
+Local = eb.VariableDifference
+Between = eb.Between
```

Comment:
I am wondering if we should get rid of `eb` altogether? Just import everything from embodied into `th`.

Also should we change `VariableDifference` to `ManifoldDifference`?

Reply:
Not sure about getting rid of `eb`, I kinda like having the separation for things like `th.eb.Collision2D`, `KinematicsModel`, and so on. 

For `VariableDifference`, maybe just `Local` and also keep the `Difference` alias?

Reply:
I meant the other way around. `VariableDifference` -> `Local` (since this is basically the manifold difference, AFAIU), and alias as `Difference` as well.

Reply:
Added.